Tests/RegressionTestsPython.cs

```diff
@@ -0,0 +1,294 @@
+/*
+ * QUANTCONNECT.COM - Democratizing Finance, Empowering Individuals.
+ * Lean Algorithmic Trading Engine v2.0. Copyright 2014 QuantConnect Corporation.
+ * 
+ * Licensed under the Apache License, Version 2.0 (the "License"); 
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at http://www.apache.org/licenses/LICENSE-2.0
+ * 
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+*/
+
+using System.Collections.Generic;
+using NUnit.Framework;
+
+namespace QuantConnect.Tests
+{
+    [TestFixture, Category("TravisExclude")]
+    public class RegressionTestsPython
+    {
+        [Test]
+        public void BasicTemplateAlgorithm()
+        {
+            AlgorithmRunner.RunLocalBacktest("BasicTemplateAlgorithm", new Dictionary<string, string>
+            {
+                {"Total Trades", "1"},
+                {"Average Win", "0%"},
+                {"Average Loss", "0%"},
+                {"Compounding Annual Return", "264.956%"},
+                {"Drawdown", "2.200%"},
+                {"Expectancy", "0"},
+                {"Net Profit", "0%"},
+                {"Sharpe Ratio", "4.411"},
+                {"Loss Rate", "0%"},
+                {"Win Rate", "0%"},
+                {"Profit-Loss Ratio", "0"},
+                {"Alpha", "0.752"},
+                {"Beta", "0.186"},
+                {"Annual Standard Deviation", "0.193"},
+                {"Annual Variance", "0.037"},
+                {"Information Ratio", "1.316"},
+                {"Tracking Error", "0.246"},
+                {"Treynor Ratio", "4.572"},
+                {"Total Fees", "$3.09"}
+            }, "Python");
```

Comment:
Perhaps its better to loop over the supported languages, and use the same result dictionary? The results should be the same for all languages.

foreach (supported language as language)
{
    RunRegressionTests (language);  
}